cvat/apps/consensus/new_intersect_merge.py

```diff
@@ -319,7 +320,13 @@ def _match_segments(
 
         return returned_values
 
-    def match_annotations_two_sources(self, a, b):
+    def match_annotations_two_sources(self, item_a: dm.Annotation, item_b: dm.Annotation) -> Tuple[
+        List[dm.Annotation],
+        List[dm.Annotation],
+        List[dm.Annotation],
+        List[dm.Annotation],
+        Optional[Dict[int, float]],
```

Comment:
This annotation means there are always 5 elements in the output tuple, which is not the same 4 or 5 elements.